ui/src/pages/authors/[id]/index.tsx

```diff
@@ -66,12 +66,12 @@ const Author: Types.NextPage<Props> = (props): React.ReactElement => {
                     </h3>
                 </header>
 
-                <section id="content" className="container mx-auto px-8 pb-12 lg:pb-24">
+                {/* <section id="content" className="container mx-auto px-8 pb-12 lg:pb-24">
```

Comment:
Maybe we remove the commented out parts if the plan is not to include it until far time post launch?